sublime-package.json

```diff
@@ -0,0 +1,309 @@
+                    },
+                    "grammarly.config.suggestions.SplitInfinitive": {
+                      "default": true,
+                      "description": "Suggests rewriting split infinitives so that an adverb doesn't come between 'to' and the verb.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.StylisticFragments": {
+                      "default": false,
+                      "description": "Suggests completing all incomplete sentences, including stylistic sentence fragments that may be intentional.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.UnnecessaryEllipses": {
+                      "default": false,
+                      "description": "Flags unnecessary use of ellipses (...).",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.Variety": {
+                      "default": true,
+                      "description": "Suggests alternatives to words that occur frequently in the same paragraph.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.Vocabulary": {
+                      "default": true,
+                      "description": "Suggests alternatives to bland and overused words such as 'good' and 'nice'.",
+                      "type": "boolean"
+                    },
+                    "grammarly.files.exclude": {
+                      "default": [],
+                      "items": {
+                        "type": "string"
+                      },
+                      "markdownDescription": "Configure [glob patterns](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) for excluding files and folders.",
+                      "order": 2,
+                      "required": true,
+                      "type": "array"
+                    },
+                    "grammarly.files.include": {
+                      "default": [
+                        "**/readme.md",
+                        "**/README.md",
+                        "**/*.txt"
+                      ],
+                      "items": {
+                        "type": "string"
+                      },
+                      "markdownDescription": "Configure [glob patterns](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) for including files and folders.",
+                      "order": 1,
+                      "required": true,
+                      "type": "array"
+                    },
+                    "grammarly.patterns": {
+                      "default": [
+                        "**/readme.md",
+                        "**/README.md",
+                        "**/*.txt"
+                      ],
+                      "description": "A glob pattern, like `*.{md,txt}` for file scheme.",
```

Comment:
This one uses markdown so should use `markdownDescription`.

Reply:
I removed that setting as it's probably not handled by the language server. I will try to reinstate it when its re-enabled. For now if someone needs to exclude a file, they should to it through the enable/disable server (Rational: I think it's better to skip the setting than giving the false impression that a file is excluded)

Reply:
That's perfectly fine. 👍